### PR TITLE
Chore/decrease ios startup time

### PR DESCRIPTION
**Pull request by siepra**

### CODE REVIEW

**Comment by siepra** [comment]
There are a few things here that still needs our attention:
1. **Leaving community stopped working**
2. **Detox test for minimizing app works but marks failure**
3. "Join/Create community" and "Username registration" screens are greying out input instead of being temporarily covered by splash screen when coming back from idle state
4. Greyed out inputs are blinking when coming back from idle state

File: packages/backend/src/nest/storage/storage.service.ts [hunk 1]
```diff
@@ -113,6 +117,57 @@ export class StorageService extends EventEmitter {
 
     this.attachFileManagerEvents()
     await this.initDatabases()
+
+    this.startIpfs()
+  }
+
+  private startIpfs() {
```
Comment (kowalski) [inline packages/backend/src/nest/storage/storage.service.ts:124]: Async

File: packages/backend/src/nest/storage/storage.service.ts [hunk 2]
```diff
@@ -113,6 +117,57 @@ export class StorageService extends EventEmitter {
...
+    this.ipfs
```
Comment (kowalski) [inline packages/backend/src/nest/storage/storage.service.ts:125]: Return

File: packages/backend/src/nest/storage/storage.service.ts [hunk 3]
```diff
@@ -113,6 +117,57 @@ export class StorageService extends EventEmitter {
 
     this.attachFileManagerEvents()
     await this.initDatabases()
+
+    this.startIpfs()
```
Comment (kowalski) [inline packages/backend/src/nest/storage/storage.service.ts:121]: Void

File: packages/backend/src/nest/storage/storage.service.ts [hunk 4]
```diff
@@ -137,12 +193,12 @@ export class StorageService extends EventEmitter {
     await this.createDbForChannels()
     this.logger('2/6')
     await this.createDbForCertificates()
-    this.logger('3/6')
-    await this.createDbForMessageThreads()
+    // this.logger('3/6')
```
Comment (kowalski) [inline packages/backend/src/nest/storage/storage.service.ts:196]: If this is no longer needed, it should be removed. If you plan to bring it back in future please add a line of comment explaining why this is commented out

File: packages/backend/src/nest/storage/storage.service.ts [hunk 5]
```diff
@@ -137,12 +193,12 @@ export class StorageService extends EventEmitter {
...
+    // await this.createDbForMessageThreads()
     this.logger('4/6')
     await this.initAllChannels()
-    this.logger('5/6')
-    await this.initAllConversations()
+    // this.logger('5/6')
```
Comment (kowalski) [inline packages/backend/src/nest/storage/storage.service.ts:200]: Same as above

**Review by kowalski — APPROVED** [review]

**Review by EmiM — APPROVED** [review]

**Pull request merged**